guides/writing-addons/index.md

```diff
@@ -16,6 +16,8 @@ ember addon <addon-name> [options]
 
 A directory called `<addon-name>` will be created containing a structure similar to an Ember app. You won't need to use all the files to make a useful addon. By convention, _most_ Ember addons start with `ember` in the name, like `ember-basic-dropdown`. This will help other developers find your addon.
 
+For more options behind this command, refer to the [CLI commands reference](https://cli.emberjs.com/release/advanced-use/cli-commands-reference/) section.
```

Comment:
If the link is pointing to somewhere in the same guides, can we use a relative path instead? (This is more pertinent if the CLI Guides can be versioned like Ember Guides.)

I think the change may be,

```suggestion
To see all options for `ember addon`, refer to the [CLI commands reference](../advanced-use/cli-commands-reference/) section.
```

I'd run the app locally to double-check.